simplq/src/components/pages/Home/CreateJoinForm.jsx

```diff
@@ -57,16 +57,18 @@ const CreateJoinForm = () => {
         />
       </div>
       <div className={styles['button-group']}>
-        <div>
-          {createInProgress ? (
-            <LoadingIndicator />
-          ) : (
-            <StandardButton onClick={handleCreateClick}>Create Queue</StandardButton>
-          )}
-        </div>
-        <div>
-          <StandardButton onClick={handleJoinClick}>Join Queue</StandardButton>
-        </div>
+        {createInProgress ? (
+          <LoadingIndicator />
+        ) : (
+          <>
+            <div>
+              <StandardButton onClick={handleCreateClick}>Create Queue</StandardButton>
+            </div>
+            <div>
+              <StandardButton onClick={handleJoinClick}>Join Queue</StandardButton>
+            </div>
+          </>
+        )}
       </div>
```

Comment:
Instead of the spinner overlapping with the join button which was pretty ugly, moved the code around a bit. Now there won't be the overlap between the spinner and button. There will still be a style change while the spinner is active because the enclosing container is flexed, so when the buttons aren't there anymore, it shrinks accordingly. I can't think of an elegant fix for this so I'll raise an issue for now.